src/winter_onboarding_2021/fleet_management_service/handlers/core.clj

```diff
@@ -0,0 +1,18 @@
+(ns winter-onboarding-2021.fleet-management-service.handlers.core
+  (:require [ring.util.response :as response]
+            [winter-onboarding-2021.fleet-management-service.db.core :as db]
+            [winter-onboarding-2021.fleet-management-service.views.core :as view]
+            [winter-onboarding-2021.fleet-management-service.views.layout :as layout]))
+
+(defn index [request]
+  (response/response (layout/application "Fleet Management Service"
+                                         (view/index))))
+
+(defn health-check [_]
+  (response/response (str {:message "Everything OK!"
+                           :data (db/healthcheck-query)})))
+
+(comment "extract body from request")
```

Comment:
These comments got committed by me by mistake in `add-cab` branch.

We could remove them to prevent merging these in `main`